packfile.h

```diff
@@ -86,6 +86,13 @@ extern void unuse_pack(struct pack_window **);
 extern void clear_delta_base_cache(void);
 extern struct packed_git *add_packed_git(const char *path, size_t path_len, int local);
 
+/*
+ * Close the pack, unlink the .pack and associated extension files.
```

Comment:
We don't actually close the pack file (in this commit at least).  We just unlink by pathname.

Reply:
thanks! I'll fix the comment.